set.mm

```diff
@@ -715873,13 +715947,15 @@ be used (following the terminology used in [Helfgott] p. 2), because there
   are a strong and a weak version of the ternary Goldbach conjecture.  The term
   _Goldbach partition_ is used for a sum of two resp. three (odd) primes
   resulting in an even resp. odd number without further specialization.
+  <br><br>
```

Comment:
two blank lines each time ?

Reply:
no, only one blank line: the first <br> performs a "carriage return", the second <br>produced an empty line.

Reply:
Curiously, the "< b r > " (without blanks) in my previous comment caused carriage returns there :-).